Wikipedia/Code/WMFSearchLanguagesBarViewController.swift

```diff
@@ -24,13 +24,21 @@ class WMFSearchLanguagesBarViewController: UIViewController, WMFPreferredLanguag
         }
     }
 
-    private var _currentlySelectedSearchLanguage: MWKLanguageLink?
     private(set) var currentlySelectedSearchLanguage: MWKLanguageLink? {
```

Comment:
Still have variable here… see line 33. 
To remove the backing store you should make this into a computed property. See:
http://www.codingexplorer.com/computed-properties-in-swift/